javascript/javascript2/week2/hyfBay.js

```diff
@@ -0,0 +1,60 @@
+  
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
+    products.forEach(product => {
+        const li = document.createElement('li');
+
+        let shipsToHTML = '';
+        product.shipsTo.forEach(country => shipsToHTML += `<li>${country}</li>`);
+
+        li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
+            </ul>
+        `;
+        productsUl.appendChild(li);
+    });
+}
+
+
+const searchBar = document.getElementById('searchBar');
+
+searchBar.addEventListener('keydown', function () {
```

Comment:
I think that using `input` event is better to use here, because it will be fired when user writes smith in an <input> field
https://www.w3schools.com/jsref/event_oninput.asp